src/pages/_offline.tsx

```diff
@@ -7,7 +7,9 @@ const Offline: NextPage = () => {
   return (
     <>
       <Head>
-        <title>Safe{'{Wallet}'} – Offline</title>
+        <title>
+          <>{'Safe{Wallet}'} – Offline</>
```

Comment:
`<title>{'Safe{Wallet} – Offline'}</title>` would also work

Reply:
I've changed all the titles accordingly.